### ISSUE TITLE
godot always crashes when i use godot beautifier theme

### ISSUE THREAD

**Issue by thewokgamur** [Vahera/godot-orchestrator#359]
godot game engine always crashed when i load visual script with godot beautifier theme when i turned off the godot beautifier theme the godot Orchestrator work perfectly fine

**Comment by Naros** [comment]
I downloaded the 2.0.0 version into Godot 4.2.2.stable, and it doesn't work.  In fact, the plugin generates a number of errors:
```
ERROR: Attempt to open script 'res://addons/godot_beautifier/src/settings_panel.gd' resulted in error 'File not found'.
   at: (modules/gdscript/gdscript.cpp:1025)
ERROR: Failed loading resource: res://addons/godot_beautifier/src/settings_panel.gd. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:274)
WARNING: res://addons/addons/godot_beautifier/src/settings_panel.tscn:4 - ext_resource, invalid UID: uid://igtp4gf8qjm8 - using text path instead: res://addons/godot_beautifier/src/logo.png
     at: load (scene/resources/resource_format_text.cpp:448)
ERROR: Resource file not found: res://addons/godot_beautifier/src/logo.png (expected type: Texture2D)
   at: (core/io/resource_loader.cpp:279)
```
So I would seem to think this is more an issue with the beautifier plugin and Orchestrator.  But if you can provide the details I requested @thewokgamur, that'd be helpful.

**Comment by Naros** [comment]
So I setup a test project usng Godot 4.3 with only godot-beautifer installed:
![image](https://github.com/Vahera/godot-orchestrator/assets/3255568/e316fd03-521b-4ab9-a7be-a49d0e4b1ad7)

This crashes Godot everytime I enable the plug-in, so I'm inclined to think the issue isn't with Orchestrator but that this plugin is doing something that is unacceptable to Godot.

**Comment by Naros** [comment]
Fixed by #390
